node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/Autoscaler.java

```diff
@@ -113,8 +105,9 @@ private boolean similar(double r1, double r2, double threshold) {
                                                           resource,
                                                           clusterNodes.stream().map(Node::hostname).collect(Collectors.toList()));
 
-        if (window.measurementCount() < minimumMeasurements) return Optional.empty();
-        if (window.hostnames() != clusterNodes.size()) return Optional.empty(); // Regulate only when all nodes are measured
+        // Require a total number of measurements scaling with the number of nodes but don't require them from all nodes
```

Comment:
The second part of this reads more like we don't require having any measurements from all nodes, instead of not requiring at least `minimumMeasurementsPerNode` for all nodes

Reply:
Better now?